assets/scss/base.scss

```diff
@@ -184,3 +184,157 @@ svg.svg-inline--fa{
 .tinymce-textarea {
   background-color: $backgroundWhite;
 }
+
+
+body {
+  font-size: .875rem;
```

Comment:
What's the reason you are mixing rem, px... Either use accessibility measures increasing with the users desire or do not.